Gemfile

```diff
@@ -51,6 +51,9 @@ gem "bootsnap", require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'rubocop', require: false
+  gem "rubycritic", require: false
+  gem  'sandi_meter'
```

Comment:
check indentation and use single quotes for strings in ruby

Reply:
i agree with @dannharo

Reply:
Done